docs/index.html

```diff
@@ -116,7 +116,7 @@ <h3><a href="https://docs.rs/commonware-reshare">reshare</a></h3>
         <p>Reshare a threshold secret over an epoched log.</p>
         <h3><a href="https://docs.rs/commonware-sync">sync</a></h3>
         <p>Synchronize state between a server and client.</p>
-        <h3><a href="https://docs.rs/commonware-vrf">vrf</a></h3>
+        <h3><a href="https://github.com/commonwarexyz/monorepo/tree/v0.0.63/examples/vrf">vrf</a></h3>
```

Comment:
Let's remove this from here (ok to keep links like you did in blogs but we don't need on homepage)